### PR TITLE
feat: support `{String}` patterns

### LINKED ISSUE #150: [Feature] support `{String}` patterns
Feature request: Could the patterns array support plain old strings, where
```javascript
patterns: [
  'myFile.txt'
]
```

is equivalent to
```javascript
patterns: [{
  from: 'myFile.txt'
}]
```

?

This would reduce boilerplate when many of the copy patterns are simple.

If this is okay, I can implement the PR.

### PR DESCRIPTION
**Pull request by cletusw**
Fixes #150

### CODE REVIEW

File: tests/index.js [hunk 1]
```diff
@@ -1053,6 +1053,25 @@ describe('apply function', () => {
         });
     });
 
+    describe('with simple string patterns', () => {
+        it('can move multiple files', (done) => {
```
Comment (cletusw) [inline tests/index.js]: Wording could be better if anyone has suggestions.

**Comment by cletusw** [comment]
@michael-ciniawsky Please review, thanks!

File: tests/index.js (hunk 1 above)
Comment (michael-ciniawsky) [reply tests/index.js]:
```js
describe('Patterns - {String}', () => { 
   it('Multiple Files', () => {})
})
```

But 🐦 , all good

**Comment by michael-ciniawsky** [comment]
@cletusw Please update the docs (`README`) aswell :)

**Comment by cletusw** [comment]
@michael-ciniawsky Okay, I've added the changes to the README to my string-patterns branch but for some reason the PR is not updating.

**Comment by cletusw** [comment]
Never mind, it just updated. Weird.

**Pull request merged**